lib/mayaUsd/render/vp2RenderDelegate/mesh.h

```diff
@@ -70,6 +70,7 @@ struct PrimvarInfo
 
     PrimvarSource                             _source;
     std::unique_ptr<MHWRender::MVertexBuffer> _buffer;
+    MFloatArray                               _extraInstanceData;
```

Comment:
It is color specific currently. I don't see we are going to have more other data coming in. So maybe rename it to `_extraInstanceColor` so its name is more informative?

Reply:
I don't like that because then if the primvar source is points it'll have a data member _extraInstanceColor with it.